Documentation/GeneralConventions/FileStructure.rst

```diff
@@ -246,17 +249,18 @@ Extension key
 The *extension-key* placeholder contains the TYPO3 extension key in case of a
 TYPO3 extension documentation.
 
-In case of any other PHP application replace the table row with
+Remove this field if the project has no extension key.
 
-.. code-block:: rst
 
-   :Package name:
-      <package-name>
+.. _index-rst-package-name:
+
+Package name
+^^^^^^^^^^^^
 
-and insert the Composer package name into the *package-name* placeholder.
+The *package-name* placeholder contains the Composer package name of the
+project.
 
-Remove this table row if the documentation does not refer to either a TYPO3
-extension or a PHP application.
+Remove this field if the project has no package name.
```

Comment:
That's basically impossible, since the package cannot be installed then.

Reply:
This guide is also for official TYPO3 manuals, like e.g. the "TCA Reference".

Reply:
Then:

> Remove this field if the project is no installable package

Reply:
Done.